nbexchange/handlers/assignment.py

```diff
@@ -322,8 +318,8 @@ def post(self):
             )
 
             if not self.request.files:
-                self.log.warning("Error: No file supplied in upload")  # TODO: improve error message
-                raise web.HTTPError(412)  # precondition failed
+                # self.log.warning("Error: No file supplied in upload")  # TODO: improve error message
```

Comment:
surely this is improving the error message so the TODO can go?

Reply:
Actually, even better - I can put the message in the HTTPError, and it's logged